tests/homework.py

```diff
@@ -31,7 +32,7 @@ def get_sum_of_corners(self, number_of_corners):
         :param number_of_corners:
         :return:
         """
-        if number_of_corners > 4:
+        if number_of_corners > 4 or number_of_corners <= 0:
```

Comment:
```suggestion
        if number_of_corners not in range(1,5):
```
старайся уникати подвійних перевірок, якщо це можливо